challenge.md

```diff
@@ -7,37 +7,37 @@ Write your answers inside this file, where it's indicated by the comments.
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
 <!-- Answer Starts Here -->
-
+git clone git@github.com:jlevinger/git-basics-08-assessment.git
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
 <!-- Answer Starts Here -->
-
+git diff
```

Comment:
Yep! We also probably want to use `git status`, since `git diff` won't show things like creating, deleting, or moving files.